packages/web-components/src/components/va-radio-option/va-radio-option.tsx

```diff
@@ -56,19 +56,24 @@ export class VaRadioOption {
 
   render() {
     const { checked, ariaDescribedby, name, value, label } = this;
+    const id = this.el.id || (name + value);
     return (
       <Host
         aria-checked={checked ? `${checked}` : 'false'}
         aria-describedby={(checked && ariaDescribedby) || null}
-        aria-label={name}
         checked={checked}
         name={name}
         onClick={() => this.handleChange()}
         role="radio"
         value={value}
-        id={this.el.id || label}
+        id={id}
       >
-        <label htmlFor={label}>{label}</label>
+        <div class="tile">
+          <label htmlFor={id} aria-describedby="description">{label}</label>
+          <div id="description">
```

Comment:
Added the `aria-describedby` pointing to the description per @joshkimux recommendations. Moving the slot inside the label may cause issues if a `va-additional-info` component is used.